docs/platform_concepts/studio/train/entities.md

```diff
@@ -3,159 +3,188 @@ title : Entities
+For instance, if you perform a fuzzy search for "rode," it will identify terms with similar spelling, such as "ride" or "node," rather than strictly requiring an exact match.
 
+6. Click **Train**.
 
-:::note
-It is highly recommended not to use text entity unless required as it is not very definite. 
-:::
 
----
+## Update entities
 
-### 4.4 System entities
+You can update the entities(all of the information in it) at any point of time. To update an entity:
 
-Apart from the List and Pattern type entities, a set of entities are recognized by the model out-of-the-box without any specific training. These include:
+1. Go to **Studio** > **Train** > **Entities**.
 
-  - Name
-  - Date
-  - Email
-  - Location
+![](https://i.imgur.com/bJNtr5w.png)
 
+2. Click **Edit**.
+3. Make the required changes and click **Update**.
 
+![](https://i.imgur.com/gktslXm.png)
 
 
+## Import entities
 
+You can import the entities from an external source. The file to be uploaded should be a CSV file with two headers, **Name** for the item name and **Synonyms** for comma-separated synonyms associated with the item.
 
----
+**Sample**
+
+<img src="https://i.imgur.com/tdhpmfW.png" alt="drawing" width="40%"/>
+
+1. Go to **Studio** > **Train** > **Entities**.
+2. Click **Import**.
+
+![](https://i.imgur.com/hYe6DO8.png)
 
-## <a name="ape"></a> 5. Autoskipping prompts using entities
+3. Click **Upload file** and upload the CSV file. 
 
-To detect entity and skip prompts, open **Make prompts smarter** of that respective prompt and select the entity value. Click [here](https://docs.yellow.ai/docs/platform_concepts/studio/build/nodes/prompt-nodes#5-make-prompt-smarter) to learn more on Make prompts smarter option. 
-If the entity value is detected/ present in the memory, that prompt will be skipped from the flow. 
+<img src="https://i.imgur.com/OWqXVb0.png" alt="drawing" width="70%"/>
 
+4. Click **Import**.
 
-> For example, if date entity is selected for autoskipping in date prompt, value of date will be automatically assigned to given date and user will not be shown that prompt. 
 
-![](https://i.imgur.com/e8gUqST.png)
+## Export entities
 
+You can export entities from our platform for backup or integration purposes. To export entities:
+
+1. Go to **Studio** > **Train** > **Entities**.
+2. Click **Export**.
+
+![](https://i.imgur.com/exJn6JX.png)
+
+Entities will be downloaded in your system as a CSV file.
 
 :::note
-You can test Entities detected in a phrase/utterance using [Test your bot](https://docs.yellow.ai/docs/platform_concepts/studio/tools#21-test-your-bot) option in tools.
-:::
\ No newline at end of file
+Import and Export actions are available only for list type entities
+:::
+
+
+## Delete entities
+
+1. Go to **Studio** > **Train** > **Entities**.
+2. Click **Delete** followed by another **Delete** button.
+
+![](https://i.imgur.com/3wgz5Nx.png)
+
+
+## Auto-skip prompts & simplify interactions with entity-based suggestions
```

Comment:
I am a bit confused on this topic... 
I was under the impression that you can skip a node when the entity is already been identified previously in a flow. 
EX: flow 1, I enter my phone number and email ID. These details are already identified as entities. 
Flow 2: There is a node that must ask for my phone number and address, but since the system already has my number, it only collects my address. 
The number node was configured to be skipped if the entity was identified. 

**Please confirm this with AK.**

Reply:
Yes, and those two use cases have already been discussed. I'm not sure about the specific question or concern you're trying to address. Could you please rephrase? Thank you!